synapseclient/models/folder.py

```diff
@@ -324,6 +370,8 @@ async def copy(
         parent_id: str,
         copy_annotations: bool = True,
         exclude_types: Optional[List[str]] = None,
+        file_update_existing: bool = False,
```

Comment:
This makes sense in a Folder context, but you can also update existing projects, tables, and other entities.

Reply:
Per this [documentation here](https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/synapseutils/copy_functions.py#L325) - It is explicit that updateExisting is only applicable in the case of a File copy. Agreed on the Table front - However, [the boolean isn't being used](https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/synapseutils/copy_functions.py#L645) during a table copy.

Reply:
Shows how much I remember for a function I wrote. 🙃